docs/modules/user-guide/partials/con_using-a-top-level-attribute.adoc

```diff
@@ -0,0 +1,38 @@
+[id="con_using-top-level-attribute_{context}"]
+= Using a top-level attribute
+
+[role="_abstract"]
+
+Use top-level attributes in devfiles to reuse global attributes in different sections of the same devfile.
+
+.Devfile variable
+
+The devfile 2.x specification includes global variables which define key value pairs.
+
+```yaml
+variables:
+  my-var: "some value"
+```
+
+Use `{{my-var}}` to reference the variables in non-string fields within the devfile. When you set only the variable field, you can make different template versions of your devfile.
+
+.Top-level attribute
```

Comment:
.Procedure 

Add a top-level attribute to define attributes as implementation-dependent, free-form yaml attributes on the top level in a devfile. 

< Procedure steps > (see my comment below)